### ISSUE TITLE
[Android] `NavigationView` doesn't initially update Flyout position

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/uno#18024]
### Current behavior

https://github.com/user-attachments/assets/d6a4865d-6dbf-4631-a16f-17437b4f4d47

### How to reproduce it (as minimally and precisely as possible)

```xml
  <NavigationView PaneDisplayMode="LeftCompact">
    <NavigationView.MenuItems>
      <NavigationViewItem Content="Home" Icon="Home" Tag="Home" />
      <NavigationViewItem Content="Sub Menu 1" Icon="Keyboard" ToolTipService.ToolTip="Sub Menu 1">
        <NavigationViewItem.MenuItems>
          <NavigationViewItem Content="Notes" Icon="Page" ToolTipService.ToolTip="Notes"/>
          <NavigationViewItem Content="Mail" Icon="Mail" ToolTipService.ToolTip="Mail"/>
          <NavigationViewItem Content="Sub Menu 2" Icon="AllApps" ToolTipService.ToolTip="Sub Menu 2">
            <NavigationViewItem.MenuItems>
              <NavigationViewItem Content="Calendar" Icon="Calendar" ToolTipService.ToolTip="Calendar"/>
              <NavigationViewItem Content="Contacts" Icon="Contact" ToolTipService.ToolTip="Contacts"/>
            </NavigationViewItem.MenuItems>
          </NavigationViewItem>
        </NavigationViewItem.MenuItems>
        </NavigationViewItem.MenuItems>
      </NavigationViewItem>
    </NavigationView.MenuItems>
    <Frame x:Name="ContentFrame" />
  </NavigationView>
```

### Works on UWP/WinUI

Yes

### Affected platforms

Android, iOS?

### IDE

Visual Studio 2022

**Comment by morning4coffe-dev** [comment]
After more investigation, actually the correct behavior seems to be that the Flyout shouldn't open at all at that point and the correct item should be selected.

**Comment by cconner100** [comment]
So why is the Hamburger icon missing at the top left of the screen?  This should open and close the menu but that option is not enabled

**Comment by morning4coffe-dev** [comment]
Thank you for the clarification, @cconner100. In the video I shared, there is a HotReload indicator that is covering the actual button. You can move it by dragging the two lines. Is that what you were referring to? 🤔

**Comment by cconner100** [comment]
Yes can drag it but the hamburger menu is still not showing in my example

From: Dominik Titl ***@***.***>
Date: Wednesday, August 28, 2024 at 6:35 PM
To: unoplatform/uno ***@***.***>
Cc: curtis conner ***@***.***>, Mention ***@***.***>
Subject: Re: [unoplatform/uno] [Android] `NavigationView` doesn't initially update Flyout position (Issue #18024)

Thank you for the clarification, @cconner100<https://github.com/cconner100>. In the video I shared, there is a HotReload indicator that is covering the actual button. You can move it by dragging the two lines. Is that what you were referring to? 🤔

**Comment by Youssef1313** [comment]
@cconner100 Could you please post a screenshot after moving the HotReload indicator away?

**Comment by cconner100** [comment]
I can move it and its still not showing but if I disable hot-reload from my source, the hamburger shows.  Currently moving the hotReload sometimes works sometimes not.  So for now the android works if I disable the hot reload

From: Youssef Victor ***@***.***>
Date: Thursday, August 29, 2024 at 5:14 PM
To: unoplatform/uno ***@***.***>
Cc: curtis conner ***@***.***>, Mention ***@***.***>
Subject: Re: [unoplatform/uno] [Android] `NavigationView` doesn't initially update Flyout position (Issue #18024)

@cconner100<https://github.com/cconner100> Could you please post a screenshot after moving the HotReload indicator away?

**Comment by Youssef1313** [comment]
The flyout issue is a pointers issue where we get an extra incorrect `Tapped` event.

```xml
	<NavigationView x:Name="NavigationView" PaneDisplayMode="LeftCompact">
		<NavigationView.MenuItems>
			<NavigationViewItem Content="Sub Menu 1" Icon="Keyboard" ToolTipService.ToolTip="Sub Menu 1">
				<NavigationViewItem.MenuItems>
					<NavigationViewItem Content="Notes" Icon="Page" ToolTipService.ToolTip="Notes"/>
				</NavigationViewItem.MenuItems>
			</NavigationViewItem>
		</NavigationView.MenuItems>
		<Frame x:Name="ContentFrame" />
	</NavigationView>
```

At the following state:

![image](https://github.com/user-attachments/assets/e2242b7d-1223-4324-b827-e0cddefa40b7)

Clicking on "Notes" will first cause Tapped event on that NavigationViewItem, which will cause the pane to be closed. Then, we get another incorrect Tapped event on "Sub Menu 1" which will cause it to open as a flyout.

**Comment by MartinZikmund** [comment]
The fix for this issue will be merged shortly, we had to confirm it is valid across all targets

**Comment by agneszitte** [comment]
@jhanvi03, @Darsh0307 the fixes will need to be tested with latest 5.6
